src/app/_components/client/Avatar/components/AvatarList/AvatarList.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import { useClickAway, useToggle } from "@/app/_hooks";
+import { UserType } from "@/app/_types/response/user";
+
+import profileImage from "../../../../../../../public/assets/userProfile.svg";
+import Avatar from "../../Avatar";
+
+import { AnimatePresence, motion } from "framer-motion";
+import Image from "next/image";
+import { twJoin } from "tailwind-merge";
+
+export interface AvatarListProps {
+  users: UserType[];
+}
+
+const AvatarList = ({ users }: AvatarListProps) => {
+  const { isShow, handleClickToggle, handleClickClose } = useToggle();
+  const ref = useClickAway<HTMLDivElement>(handleClickClose);
+
+  return (
+    <>
+      <div
+        className="relative"
+        ref={ref}
+      >
+        <div
+          className="inline-flex cursor-pointer items-center"
+          onClick={handleClickToggle}
+        >
+          {users.slice(0, 2).map(({ userId, userName, userImage }, index) => (
+            <div
+              key={userId}
+              className={twJoin(
+                "relative h-[3rem] w-[3rem] overflow-hidden rounded-full",
+                index === 1 && "-ml-[2.2rem] mt-[1rem]",
+                index === 0 && "z-10",
+              )}
+            >
+              <Image
+                className="object-cover"
+                src={userImage || profileImage}
+                alt={userName}
+                sizes="100%"
+                priority
+                fill
+              />
+            </div>
+          ))}
+        </div>
+        {isShow && (
+          <AnimatePresence>
+            <motion.ul
+              className="absolute left-0 top-0 z-10 flex"
+              initial={{ x: "-100%" }}
+              animate={{ x: 0 }}
+              exit={{ x: "-100%" }}
+              whileTap={{ scale: 0.95 }}
+            >
+              {users.map((user) => (
+                <li
+                  className="mr-2 overflow-hidden rounded-full"
+                  key={user.userId}
+                >
+                  <Avatar user={user} />
+                </li>
+              ))}
+            </motion.ul>
+          </AnimatePresence>
+        )}
```

Comment:
[ P1 ]

해당 부분에서 Exit 동작하게 하시려면

```suggestion
        <AnimatePresence>
          {isShow && (
            <motion.ul
              className="absolute left-0 top-0 z-10 flex"
              initial={{ x: "-100%" }}
              animate={{ x: 0 }}
              exit={{ x: "-100%" }}
              whileTap={{ scale: 0.95 }}
            >
              {users.map((user) => (
                <li
                  className="mr-2 overflow-hidden rounded-full"
                  key={user.userId}
                >
                  <Avatar user={user} />
                </li>
              ))}
            </motion.ul>
          )}
        </AnimatePresence>
```

위와같이 `AnimatePresence` 태그 내부에서 상태값에 의해 언마운트 마운트가 발생해야합니다!

Reply:
넵 !! 감사합니다 !! ㅎㅎ 👏😊
`AnimatePresence` 내부 자식이 언마운트 시에 발동되는군요 ?!